templates/motd.erb

```diff
@@ -1,3 +1,6 @@
+### File managed with puppet ###
+## Module: '<%= scope.to_hash['module_name'] %>'
+
```

Comment:
Rather than testing for this (which you would see on every login), why don't we test with `grep 'property of the Foreman project'` instead? It's highly unlikely that's in an out of the box motd so a good test.